content/src/fundings/clean-energy-program-combined-heat-and-power-fuel-cells.md

```diff
@@ -1,5 +1,5 @@
 ---
-filename: New Jersey’s Clean Energy Program – Combined Heat & Power - Fuel Cells
+filename: Clean Energy Program - Combined Heat and Power - Fuel Cells
```

Comment:
[dust] just wanna make sure this & vs and change was inteitional and not part of something automated

Reply:
Yep, intentional!